ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckWithCommittedImpl.scala

```diff
@@ -149,4 +158,66 @@ private[validate] class ConflictCheckWithCommittedImpl(
     }
   }
 
+  private def sameContractData(
+      actual: (Value.VersionedContractInstance, Timestamp),
+      provided: DisclosedContract,
+  )(implicit loggingContext: LoggingContext): Boolean = {
+    // TODO DPP-1026: Note: Not validating transaction version or agreement text here, because we're making those up
+
+    // TODO DPP-1026: Template id validation can probably be removed once we switch from VersionedContractInstance to Value
+    val actualTemplate = actual._1.unversioned.template
+    val providedTemplate = provided.contractInst.unversioned.template
+
+    // TODO DPP-1026: Make sure to compare the whole argument
+    val actualArgument = actual._1.unversioned.arg
+    val providedArgument = provided.contractInst.unversioned.arg
+
+    if (actualTemplate != providedTemplate) {
+      // TODO DPP-1026: fix the logging context, it should include at least the submission id (to track malicious users).
```

Comment:
i guess not super crucial given that sox is only a test ledger but still seems sensible to fix eventually.

Reply:
My intention is that all `// TODO DPP-1026:` comments need to be resolved before this can be merged to master. This one is definitely something that should be resolved for a production ready participant.

Reply:
my point was that this seems like sox code not participant code. So a production ready Canton or vmbc participant won’t use this.